samples/inputs/date-time-input/input-format-display-format/README.md

```diff
@@ -0,0 +1,56 @@
+<!-- NOTE: do not change this file because it's auto re-generated from template: -->
+<!-- https://github.com/IgniteUI/igniteui-react-examples/tree/vnext/templates/sample/ReadMe.md -->
+
+This folder contains implementation of React application with example of Sizing feature using [Icon](https://www.infragistics.com/products/ignite-ui-react/react/components/general-getting-started.html) component.
```

Comment:
All the `README.md` files seem to be related to the `Icon` component. They should update automatically when [running all samples](https://github.com/IgniteUI/igniteui-react-examples?tab=readme-ov-file#running-all-samples).